docs/user/build/install/jdk.rst

```diff
@@ -81,8 +42,6 @@ RedHat OpenJDK            GPL                 x                            2024
 Adopt OpenJDK             GPL                 x     x              x       2022 September
 ========================= =================== ===== ====== ======= ======= ==============
 
-.. warning:: Since the API changes from Java version to version, building a GeoTools version with a newer Java SDK is risky (you may accidentally use a new method). Pull requests are tested against Java 8, but we do ask you to be careful.
```

Comment:
This warning still has value; we can remove the sentence about testing pull requests  against Java 8, but as far as I know compiling with Java 17 may still accidentally reference new methods or changed method signatures resulting in byte code incompatibility.

Reply:
Yes indeed, but we do not officially support Java 17, only in an experimental way. We need a Java 17 specific section, the warning should not be in the Java 11 one. Would you like to add some wording about it?